src/plugins/ucx/ucx_backend.cpp

```diff
@@ -1388,6 +1388,35 @@ nixlUcxEngine::getWorkerId() const {
     return it->second;
 }
 
+[[nodiscard]] std::optional<size_t>
```

Comment:
```suggestion
std::optional<size_t>
```
The attribute is used only during declaration, and is not needed during definition.